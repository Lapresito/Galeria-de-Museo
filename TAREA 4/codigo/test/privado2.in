crearVisitaDia 01/01/2020
fechaVisitaDia
maxGruposVisitaDia
cantidadGruposVisitaDia
crearGrupoABB 
crearVisitante 1 Juan Gomez 21
agregarAGrupoABB
crearVisitante 2 Jorge Jimenez 20
agregarAGrupoABB
encolarGrupoVisitaDia 
cantidadGruposVisitaDia
imprimirVisitaDia
masPrioritarioVisitaDia 
estaEnVisitaDia 1
estaEnVisitaDia 2
prioridadVisitaDia 1
crearGrupoABB 
crearVisitante 3 Peter Parker 22
agregarAGrupoABB
crearVisitante 4 Heber Everton 27
agregarAGrupoABB
encolarGrupoVisitaDia
estaEnVisitaDia 1
estaEnVisitaDia 2
cantidadGruposVisitaDia
prioridadVisitaDia 1
prioridadVisitaDia 2
imprimirVisitaDia
masPrioritarioVisitaDia 
invertirPrioridadVisitaDia
masPrioritarioVisitaDia 
liberarVisitaDia
Fin
