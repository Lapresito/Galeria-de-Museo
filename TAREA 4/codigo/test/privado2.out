Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> La visita dia se ha creado de forma exitosa.
2> La visitaDia corresponde a la fecha: 1/1/2020
3> Pueden existir 100 max grupos en la visitaDia.
4> La cantidad de grupos en la visita dia es 0
5> El grupoABB ha sido creado de forma exitosa.
6> Visitante creado de forma exitosa.
7> Se ha agregado el visitante al grupo de forma exitosa.
8> Visitante creado de forma exitosa.
9> Se ha agregado el visitante al grupo de forma exitosa.
10> Se ha agregado el grupo a la visita dia de forma exitosa.
11> La cantidad de grupos en la visita dia es 1
12> Visita para dia: 1/1/2020

Nivel 1
1) Grupo 1 con edad promedio 20.50
Visitante Juan Gomez
Id: 1
Edad: 21
Visitante Jorge Jimenez
Id: 2
Edad: 20
13> El grupo con mayor prioridad de la visitaDia es:
Visitante Juan Gomez
Id: 1
Edad: 21
Visitante Jorge Jimenez
Id: 2
Edad: 20
14> El grupo de id 1 pertenece a la visitaDia.
15> El grupo de id 2 NO pertenece a la visitaDia.
16> La prioridad del grupo de id 1 es 20.50 .
17> El grupoABB ha sido creado de forma exitosa.
18> Visitante creado de forma exitosa.
19> Se ha agregado el visitante al grupo de forma exitosa.
20> Visitante creado de forma exitosa.
21> Se ha agregado el visitante al grupo de forma exitosa.
22> Se ha agregado el grupo a la visita dia de forma exitosa.
23> El grupo de id 1 pertenece a la visitaDia.
24> El grupo de id 2 pertenece a la visitaDia.
25> La cantidad de grupos en la visita dia es 2
26> La prioridad del grupo de id 1 es 20.50 .
27> La prioridad del grupo de id 2 es 24.50 .
28> Visita para dia: 1/1/2020

Nivel 1
1) Grupo 1 con edad promedio 20.50
Visitante Juan Gomez
Id: 1
Edad: 21
Visitante Jorge Jimenez
Id: 2
Edad: 20

Nivel 2
2) Grupo 2 con edad promedio 24.50
Visitante Peter Parker
Id: 3
Edad: 22
Visitante Heber Everton
Id: 4
Edad: 27
29> El grupo con mayor prioridad de la visitaDia es:
Visitante Juan Gomez
Id: 1
Edad: 21
Visitante Jorge Jimenez
Id: 2
Edad: 20
30> Se ejecuto invertirPrioridadTVisitaDia exitosamente.
31> El grupo con mayor prioridad de la visitaDia es:
Visitante Peter Parker
Id: 3
Edad: 22
Visitante Heber Everton
Id: 4
Edad: 27
32> La visita dia se ha liberado exitosamente.
33> Fin.
